scripts/storage-emulator-integration/internals.test.ts

```diff
@@ -126,7 +126,7 @@ describe("Emulator Internals", function (this) {
     const smallFilePath = createRandomFile("testFile", SMALL_FILE_SIZE);
     await testGcsBucket.upload(smallFilePath, { destination: "public/testFile" });
 
-    const downloadUrl = await page.evaluate(async () => {
```

Comment:
This was getting flagged by lint for me (async without an await), so I'm cleaning it up even tho it is unrelated.